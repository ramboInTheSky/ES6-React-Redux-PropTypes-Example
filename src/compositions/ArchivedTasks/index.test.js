import React from 'react';
import { shallow } from 'enzyme';

import { ArchivedTasksComposition } from './';

describe('<ArchivedTasksComposition />', () => {
  let props;
  let el;

  beforeEach(() => {
    props = {
      arrearsDetailHeading: 'Arrears Details',
      arrearsId: '123',
      updatePageHeader: jest.fn(),
    };

    el = shallow(<ArchivedTasksComposition {...props} />);
  });

  it('should render the page', () => {
    expect(el).toMatchSnapshot();
  });

  it('should update the header', () => {
    expect(props.updatePageHeader).toHaveBeenCalled();
  });
});
