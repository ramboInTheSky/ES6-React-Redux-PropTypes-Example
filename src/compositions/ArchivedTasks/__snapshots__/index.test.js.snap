// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ArchivedTasksComposition /> should render the page 1`] = `
<withRouter(Connect(PageContentComposition))
  ignoreArrearFetch={true}
>
  <div
    className="col-lg-9"
  >
    <Styled(styled.div)>
      <WithTheme(withRouter(Connect(ArrearsDetailTasksComposition)))
        archived={true}
      />
    </Styled(styled.div)>
  </div>
  <aside
    className="col-lg-3"
  >
    <WithTheme(Button)
      data-bdd="ArchivedTasks-arrearsDetailLink"
      href="/arrears-details/123"
      isFullWidth={true}
      to="/arrears-details/123"
    >
      Arrears Details
    </WithTheme(Button)>
  </aside>
</withRouter(Connect(PageContentComposition))>
`;
